lib/wallet/test/unit/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -1265,9 +1270,15 @@ spec_updateTx = describe "updateTx" $ do
             -- Could be argued that it should instead return `Right tx`.
             let anyRecentEraTx = recentEraTxFromBytes
                     $ snd $ head signedTxs
-            Write.withInAnyRecentEra anyRecentEraTx $ \tx ->
-                updateTx tx noTxUpdate
-                    `shouldBe` Left (ErrExistingKeyWitnesses 1)
+            Write.withInAnyRecentEra anyRecentEraTx
+                $ \(tx :: Cardano.Tx era) -> do
+                    let res = toCardanoTx @era
```

Comment:
The new need for type-applications is very ugly, but think we can live with it for a while